tests/features/rhpam/businesscentral-monitoring/rhpam-businesscentral-monitoring.feature

```diff
@@ -46,3 +46,26 @@ Feature: RHPAM Business Central Monitoring configuration tests
      And container log should contain -Dorg.kie.server.controller.openshift.prefer.kieserver.service=true
      And container log should contain -Dorg.kie.server.controller.template.cache.ttl=10000
      And container log should contain -Dorg.kie.controller.ping.alive.disable=true
+
+  Scenario: Verify if the properties were correctly set using DEFAULT MEM RATIO
+    When container is started with args
+      | arg       | value                                                    |
+      | mem_limit | 1073741824                                               |
+      | env_json  | {"JAVA_MAX_MEM_RATIO": 80, "JAVA_INITIAL_MEM_RATIO": 25} |
+    Then container log should match regex -Xms205m
+     And container log should match regex -Xmx819m
+
+  Scenario: Verify if the DEFAULT MEM RATIO properties are overrided with different values
+    When container is started with args
+      | arg       | value                                                    |
+      | mem_limit | 1073741824                                               |
+      | env_json  | {"JAVA_MAX_MEM_RATIO": 25, "JAVA_INITIAL_MEM_RATIO": 50} |
+    Then container log should match regex -Xms128m
```

Comment:
There is inittial mem ration set higher then max mem, is that a typo or it is intended for testing?  If this is for testing, please update scenario description.

Reply:
Was a friday evening typo, fixed